Makefile

```diff
@@ -48,6 +48,13 @@ PROJECT_SLUG := github.com/vmware-tanzu/vm-operator
 # ROOT_DIR_IN_GOPATH is non-empty if ROOT_DIR is in the GOPATH.
 ROOT_DIR_IN_GOPATH := $(findstring $(GOPATH)/src/$(PROJECT_SLUG),$(ROOT_DIR))
 
+# R_WILDCARD recursively searches for a file that matches $1.
```

Comment:
Matches $2 in $1 no?

Reply:
Good catch. Will update the comment to clarify.